src/ape/managers/project/types.py

```diff
@@ -194,8 +194,15 @@ def create_manifest(
                 name=self.name,
                 version=self.version,
             )
+            del manifest.contract_types
```

Comment:
I think [this function](https://github.com/ApeWorX/ape/pull/1239/files#diff-e56910138250dd5c83fc082fb33e1acd230a5f00b386aab6405099d768b1c09bR171-R173) likely needs to be rewritten 

Ultimately, what we want is for dependencies to be fetched/compiled as manifests and disassembled as needed for individual files (for imports), and the local project to compile contract types as individual files under the contract types folder, then assemble the manifest if/when you publish the manifest (which the framework doesn't support... _yet_)

Reply:
will you do the rewrite and accept this one ?

Reply:
`ApeProject.create_manifest` should basically do this (for a local dev project):
1. fetch sources from `contracts/`
2. fetch checksum information, etc. from `.build/__local__.json`
3. fetch contract types from `.build/<ContractTypeName>.json`
4. fetch metadata from `ape-config.yaml`
5. combine into `PackageManifest` type

Reply:
thanks for steps @fubuloubu but to me when i look into the code, this is already there, so i am more confused right now.

Reply:
@fubuloubu are the above changes really necessary for @bilbeyt's PR to be merged?

From what I can tell, his PR fixes a real issue for us and would allow us to proceed with our move to ape.
While we definitely want to help improve ape and have already demonstrated that, we do not have unlimited time to do so and have to balance our priorities.

Apart from this issue, there's still a big blocker for our move, namely the pretty bad performance compared to
what we have with brownie. In that light, I would like to ask you to reconsider whether requiring additional
work here is really reasonable.

Reply:
If you address my small comments below, we can merge this and I can handle refactors later.

Reply:
@istankovic I was talking a bit more out loud to describe the overall vision, we have this tracked elsewhere. @bilbeyt's change is basically good, just wanted to make sure he can see what the design was supposed to fit in to